docs/SetupDatabaseForAWS.md

```diff
@@ -21,3 +21,10 @@ Note:-
 
 * Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) to create an AWS RDS MySQL/PostgreSQL/Oracle/SQL Server database instance.
 * (Optional) Configure advanced monitoring services with the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Monitoring.html), monitoring is enabled by default on AWS RDS database instances.
+
+## Amazon Aurora for MySQL and PostgreSQL
```

Comment:
```suggestion
## Amazon Aurora MySQL and Amazon Aurora PostgreSQL
```
It's not `for` MySQL. Just a MySQL-compatible Amazon Aurora.

Reply:
Understood. Thank you for the correction. 
I have updated the section title as suggested.